bitnami/metrics-server/templates/deployment.yaml

```
@@ -14,6 +14,9 @@ spec:
       {{- end }}
     spec:
 {{- include "metrics-server.imagePullSecrets" . | indent 6 }}
+    {{- if .Values.priorityClassName }}
+      priorityClassName: "{{ .Values.priorityClassName }}"
+    {{- end }}
```

Could you indent the block so it's aligned with the text?
```suggestion
      {{- if .Values.priorityClassName }}
      priorityClassName: {{ .Values.priorityClassName | quote }}
      {{- end }}
```